go.mod

```diff
@@ -216,3 +216,5 @@ replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-d
 go 1.23.6
 
 retract v0.0.2
+
+replace github.com/lightningnetwork/lnd/kvdb => ./kvdb
```

Comment:
not sure if the linter will complain because of that.

Reply:
Think we need to upgrade the `kvdb`, tag it and update the version here?

Reply:
or merge this one, tag kvdb, and remove this entry ?